src/app/brew/brew-add/brew-add.component.ts

```diff
@@ -201,6 +203,39 @@ export class BrewAddComponent implements OnInit {
       this.uiToast.showInfoToast('TOAST_BREW_ADDED_SUCCESSFULLY');
     }
 
+    if ('reporting_endpoint' in this.settings.reporting_parameters && this.settings.reporting_parameters.reporting_endpoint !== '') {
```

Comment:
I'd prefer if this data-reporting/analyses would be excluded to an own service.
Like uiReporting.ts below /services/ so you can call 
uiReporting.reportAnalyticsData(this.data) etc. and we can work more easy with this.
So the brew-edit need to be reworked again for this aswell